src/GrampsJs.js

```diff
@@ -729,6 +743,14 @@ export class GrampsJs extends LitElement {
   }
 
   _loadStrings (strings, lang) {
+    // First load the strings for the current language
+    if (lang in additionalStrings) {
+      this._strings = additionalStrings[lang]
+      this._strings.__lang__ = lang
+      this._lang = lang
+    }
+
+    // Then load the strings from the server and merge them with the current language strings
```

Comment:
If I see it correctly, you are adding the additionalStrings twice, no?

So far this method was doing the API call for the backend strings and, after it succeeded, added the additonalStrings. Now we want a logic that allows already using the additionalStrings before we can make an API call. I see two options to implement it: either using the `if ('error' in data)` part of the fetch handler. Currently, it shows an error message (`_showError`) but we could also say if there is an error (e.g. since we are not authorized yet), just load the additionalStrings. The second possibility would be to do something like the `this.loadingState > LOADING_STATE_UNAUTHORIZED_NOCONNECTION` that is done in the `update` method: only fetch the backend strings if we know we are authorized, otherwise just load the additionalStrings.